packages/react-native-editor/__device-tests__/pages/editor-page.js

```diff
@@ -28,7 +28,24 @@ const {
 const initializeEditorPage = async () => {
 	const driver = await setupDriver();
 	await isEditorVisible( driver );
-	return new EditorPage( driver );
+	const initialValues = await setupInitialValues( driver );
+	return new EditorPage( driver, initialValues );
+};
+
+const ADD_BLOCK_ID = isAndroid()
+	? 'Add block, Double tap to add a block'
+	: 'Add block';
```

Comment:
Nitpick: Since this is a constant I'd consider having it before the definition of functions.